src/__stories__/Home.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+//components
+import HomePage from "@/pages/index";
+import Layout from "@/layouts/Layout";
```

Comment:
create a folder "mainLayout" and create index file for layout

Reply:
done